scripts/initialChecks.js

```diff
@@ -0,0 +1,33 @@
+const path = require('path')
+require('dotenv').config({
+  path: path.join(__dirname, '../.env')
+})
+const Web3 = require('web3')
+const ERC677BridgeTokenABI = require('../abis/ERC677BridgeToken.abi')
+const { ERC_TYPES } = require('../src/utils/constants')
+
+async function initialChecks() {
+  const { ERC20_TOKEN_ADDRESS, BRIDGE_MODE, FOREIGN_RPC_URL, FOREIGN_BRIDGE_ADDRESS } = process.env
+  const result = {}
+
+  if (BRIDGE_MODE === 'ERC_TO_ERC' && process.argv[2].includes('affirmation-request-watcher')) {
```

Comment:
What benefit to specify that the json structure must be generated for `affirmation-request-watcher` only is? It seems that this script too tied to the rest of functionality, so if the watcher name is changed, the changes in this file must be done as well. What do you think?

Reply:
The idea was to avoid other watchers to do those checks since they don't need it. But I agree with you that this is too tied and it won't have a big impact on other watchers. Updated here https://github.com/poanetwork/token-bridge/pull/145/commits/f2610a95161c55007c69f0ff34d3788ac69f22f2